src/schema/objects/metadata.yaml

```diff
@@ -3014,6 +3190,26 @@ SEEGChannelCount:
     Number of SEEG channels.
   type: integer
   minimum: 0
+SampleCoordinateSystem:
+  name: SampleCoordinateSystem
+  display_name: Coordinate System of the Gaze Position
+  description: |
+    Coordinate system of the gaze position recordings.
+    Generally eye-tracker are set to use `"gaze-on-screen"` coordinate system but you may use
+    `"eye-in-head"` or `"gaze-in-world"` or other alternatives of your choice.
+    If you use the standard `"gaze-on-screen"`, it is RECOMMENDED to use this
```

Comment:
I would make this an enum and allow "custom" so the user can specify a system that is not captured by the enum.